templates/santa/application/participants_discord.html.twig

```diff
@@ -13,18 +13,25 @@
 {% endmacro %}
 
 {% block user_item %}
+    {% set user = userForm.parent.vars.choices[userForm.vars.value].data %}
+
     <label
             class="user-item"
-            for="user-{{ user.identifier }}"
+            for="{{ userForm.vars.id }}"
             data-search-index="{{ user.name ~ ' ' ~ user.extra.nickname }}"
     >
-        <input type="checkbox"
-               {{ user.identifier in selectedUsers ? 'checked="checked"' : '' }}
-               name="users[]"
-               value="{{ user.identifier }}"
-               id="user-{{ user.identifier }}"
-               data-summary="{{ current.userSummary(user)|escape('html_attr') }}"
-        >
+
+        {{ form_widget(userForm, (user.identifier in selectedUsers ?
+            {'attr': {
+                'checked': 'checked',
+                'data-summary': current.userSummary(user)|escape('html_attr'),
```

Comment:
I'm not sure if the `html_attr` is needed here (maybe symfony already does it?)

Reply:
I kept it because it was the old code, but it indeed isn't required anymore

Reply:
Actually, removing it causes the data to be treated like html and to appear on the page in the middle of the labels. I do not have any solutions to this for now, except for leaving it